### PR TITLE
Amarja Selenium Assignment1 Login & Registration

### PR DESCRIPTION
**Pull request by AmarjaBTechnocredits**
Mayur & Vishal please peer review and let me know for review comments.Awaiting approval by Friday EOD 08-Mar

### CODE REVIEW

File: seleniumByTechnocredits/src/seleniumPractise/amarja/Homework1.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import util.Init;
+
+public class Homework1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/index.html#");
+		driver.findElement(By.partialLinkText("Elements")).click();
+		Thread.sleep(1000);
+		driver.findElement(By.xpath("//input[contains(@id,'UserFirst')]")).sendKeys("Amarja");
```
Comment (Mayur-Suryawanshi1992) [inline seleniumByTechnocredits/src/seleniumPractise/amarja/Homework1.java]: Instead of hardcode value, Please use values from a properties file.

**Review by VishalKTechnocredits — COMMENTED** [review]
As Mayur Suggested use values from property file. Otherwise code is ok for me.

File: seleniumByTechnocredits/src/seleniumPractise/amarja/Homework2.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+/*Program to automated the Alert and 2) Alert > Ok
+Validate the string of Alert with hardcoded value.*/
+public class Homework2 {
```
Comment (VishalKTechnocredits) [inline seleniumByTechnocredits/src/seleniumPractise/amarja/Homework2.java:13]: Code looks good to me.

File: seleniumByTechnocredits/src/seleniumPractise/amarja/Homework2.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/index.html#");
+		Thread.sleep(1000);
+		// Finding xpath using partial link text also using xpath
+		// driver.findElement(By.partialLinkText("Elements")).click();
+		driver.findElement(By.xpath("//a[@id='basicelements']")).click();
+		System.out.println("Clicked on Basic elements");
+		// WebElement e1=
+		// driver.findElement(By.xpath("//button[starts-with(@id,'javascriptAlert')]"));----Xpath
```
Comment (Mayur-Suryawanshi1992) [inline seleniumByTechnocredits/src/seleniumPractise/amarja/Homework2.java]: Please remove commented code..

Rest of code Looks good to me

**Review by Mayur-Suryawanshi1992 — APPROVED** [review]

File: seleniumByTechnocredits/src/seleniumPractise/amarja/Homework1.java (hunk 1 above)
Comment (AmarjaBTechnocredits) [reply seleniumByTechnocredits/src/seleniumPractise/amarja/Homework1.java]: Removed hard coded values,Included code to read values from property files

**Comment by AmarjaBTechnocredits** [comment]
Code review comments incoprorated .Received Approval .Krishna please review

**Review by KrishnaKTechnocredits — APPROVED** [review]
Please do not write the whole code in the main method instead of Please create a separate method for that and call that one method in the main method. 
Rest of all the code is very clean to review and understand, Good job Amarja !!

**Pull request merged**